content/en/monitors/create/types/composite.md

```diff
@@ -140,6 +140,22 @@ Consider a composite monitor that uses two individual monitors: `A` and `B`. The
 
 **Note**: When the composite has `notify_no_data` to false, and the result of the evaluation of the sub-monitors should end up on a `No Data` status for the composite, the composite uses the last known state instead.
 
+### Composites and Downtimes
+
+A composite monitor and its individual monitors are independent of each other.
+
+**Downtime on composite monitor**
+
+Take a composite monitor that consists of two individual monitors with the condition `A \|\| B`. Creating a downtime on the composite monitor will surpress notifications from said composite only.
+
+In case monitor `A` or monitor `B` notify services or teams as well in their respective monitor configurations, the downtime on the composite alone will not mute any notifications caused by them. To achieve this, a separate downtime on the composite is needed.
+
+**Downtime on an individual monitor used in a composite monitor**
+
+Using the example above above a composite monitor that consists of two individual monitors with the condition `A \|\| B`. Creating a downtime on the individual monitor `A` will not mute the composite monitor.
```

Comment:
"Using the example above": this behaviour doesn't depend on the query at all (`A && B` would do the same thing for instance). This might be confusing as one could think that this only is the case because A notifying or B notifying means that even if one is muted the other will make the composite notify.